### DISCUSSION TITLE
Workflow support for automation between Issues and Linked Pull Requests

### DISCUSSION THREAD

**Discussion by polevaultweb** [community/community#5906] [Projects and Issues]
We are using the new project boards only with issues added to them, but the status of issues can be affected by the state of linked pull requests. It would be awesome if the (custom) workflows would be able to work between these relationships.

Eg. 

When a PR (not necessarily an item on the board) 
Has a label

Set (linked) Issue 
To a status

For example, if a pull request has been reviewed but has changes requested, I want the issue linked to the PR to move to a different status

**Comment by achimnol** [comment]
I bump up the idea, and here is an elaboration of the idea.

When there are multiple PRs linked to an issue, there should be an aggregation logic such as:
 - Set the status of the linked issue to XXX if the statuses of all linked PRs become XXX. (e.g., `In Progress` -> `Done`)
 - Set the status of the linked issue to YYY if the status of one of linked PRs become YYY. (e.g., `In Progress` <-> `Under Review`)
